java/serve/src/main/java/io/ray/serve/router/ReplicaSet.java

```diff
@@ -55,18 +70,15 @@ public ReplicaSet(String deploymentName) {
   @SuppressWarnings("unchecked")
   public synchronized void updateWorkerReplicas(Object actorSet) {
     List<String> actorNames = ((ActorNameList) actorSet).getNamesList();
-    Set<ActorHandle<RayServeWrappedReplica>> workerReplicas = new HashSet<>();
+    Set<BaseActorHandle> workerReplicas = new HashSet<>();
```

Comment:
Why do use ActorHandle instead of ActorId? or at least, we should use `HashMap<ActorId, BaseActorHandle>`. 

My concern is that ActorHandle is not used as hash key.

Reply:
Resolved. Use the actor name as the map key.